config/initializers/wrap_parameters.rb

```diff
@@ -5,10 +5,10 @@
 
 # Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
 ActiveSupport.on_load(:action_controller) do
-  wrap_parameters format: [:json] if respond_to?(:wrap_parameters)
```

Comment:
Do you know what this does? (I don't)

Reply:
I think [these docs](http://api.rubyonrails.org/v5.1/classes/ActionController/ParamsWrapper.html) explain it very well. If I understand this correctly these is only interesting if we build a webservice, which deals with corresponding requests? The extra check with respond_to is a small mystery to me. Does this imply that there are ActionController, which doesn't implement this method?

Reply:
Hmm, I don't know. I think it's good like it is now (without the if statement), because it works fine and is the default in Rails 5.